protocols/kad/src/handler.rs

```diff
@@ -90,7 +91,15 @@ pub struct KademliaHandler<TUserData> {
     next_connec_unique_id: UniqueConnecId,
 
     /// List of active outbound substreams with the state they are in.
-    outbound_substreams: SelectAll<OutboundSubstreamState<TUserData>>,
+    active_outbound_substreams: SelectAll<OutboundSubstreamState<TUserData>>,
```

Comment:
I get the idea of renaming this but I'd prefer sticking with the old name for consistency with `inbound_substreams`.